spec/index.html

```diff
@@ -503,6 +503,10 @@ <h3>RDF Documents and Syntaxes</h3>
 </section>
 
 <section id="conformance">
+  <p id="advisement-levels">The key words “strongly encouraged”, “strongly
+  discouraged”, “encouraged", “discouraged", “can", “cannot”, “could”, “could
```

Comment:
Nitpick: Let's be consistent about the quotation marks used ;-)
```suggestion
  discouraged”, “encouraged”, “discouraged”, “can”, “cannot”, “could”, “could
```